### PR TITLE
PAINTROID-141 No feedback where images are saved or copied to

### PR DESCRIPTION
**Pull request by CodeChamp-SS**
Added the place in the file system where the image or copy was saved in the bubble.

https://jira.catrob.at/browse/PAINTROID-141

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [ ] Include the name of the Jira ticket in the PR’s title
- [ ] Include a summary of the changes plus the relevant context
- [ ] Choose the proper base branch (*develop*)
- [ ] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [ ] Perform a self-review of the changes
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [ ] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [ ] Stick to the project’s gitflow workflow
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by wslany — CHANGES_REQUESTED** [review]
Nice. 

Please fix the lint warnings https://jenkins.catrob.at/job/Paintroid/job/PR-870/1/analysis/

Lots of post-save and post-copy tests are failing now, see https://jenkins.catrob.at/job/Paintroid/job/PR-870/1/testReport/

Also, please add an automatic test that checks the new behavior.

On Android 6.0 on a Samsung Galaxy S7, no destination is shown:
image

Same also on Android 7.1 and Android 8, both on a Google Pixel phone, as well as Android 9 on a Pixel 3a XL, no path information is shown in the toast:
image
image
image

It seems to work only on Android 10 and 11.

File: Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.java [hunk 1]
```diff
@@ -782,9 +804,9 @@ public void onSaveImagePostExecute(@SaveImageRequestCode int requestCode, Uri ur
 		}
 
 		if (saveAsCopy) {
-			navigator.showToast(R.string.copy, Toast.LENGTH_LONG);
+			Toast.makeText(fileActivity, context.getString(R.string.copy) + " to\n" + getRealPathFromURI(fileActivity, uri), Toast.LENGTH_LONG).show();
 		} else {
-			navigator.showToast(R.string.saved, Toast.LENGTH_LONG);
+			Toast.makeText(fileActivity, context.getString(R.string.saved) + " to\n" + getRealPathFromURI(fileActivity, uri), Toast.LENGTH_LONG).show();
 			model.setSavedPictureUri(uri);
```
[the remark this answers is not in the document]
Comment (CodeChamp-SS) [reply Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.java:810]: ok

**Comment by CodeChamp-SS** [comment]
> Nice.
> 
> Please fix the lint warnings https://jenkins.catrob.at/job/Paintroid/job/PR-870/1/analysis/
> 
> Lots of post-save and post-copy tests are failing now, see https://jenkins.catrob.at/job/Paintroid/job/PR-870/1/testReport/
> 
> Also, please add an automatic test that checks the new behavior.
> 
> On Android 6.0 on a Samsung Galaxy S7, no destination is shown:
> image
> 
> Same also on Android 7.1 and Android 8, both on a Google Pixel phone, as well as Android 9 on a Pixel 3a XL, no path information is shown in the toast:
> image
> image
> image
> 
> It seems to work only on Android 10 and 11.

hmm, I'l try to fix the issues soon.

**Comment by CodeChamp-SS** [comment]
I think now the failing tests should pass, and I've also added tests for checking the new behavior.

**Comment by CodeChamp-SS** [comment]
@wslany sir, can you please recheck the new behavior on different versions of android? I hope it should work fine now : )

**Comment by CodeChamp-SS** [comment]
I reformatted the whole MainActivityPresenter Class using android studio's code formatting functionality, but still, it's showing lint warnings, I think there is some problem with the Jenkins static analysis tool 🤔

**Comment by CodeChamp-SS** [comment]
@wslany sir, now it doesn't show any new warnings and I hope it should work fine now for different versions of android as well. Can you please review it once again?

**Comment by wslany** [comment]
🙏 @CodeChamp-SS could you cross-review @chbinder 's #857 PR which is also in code review? It is just one line further, see https://jenkins.catrob.at/job/Paintroid/job/PR-870/8/testReport/org.catrobat.paintroid.test.espresso/MenuFileActivityIntegrationTest/Tests___Device_Tests___testSaveCopy/ but actually is already occurring long before your pull request.

**Comment by wslany** [comment]
> 🙏 @CodeChamp-SS could you cross-review @chbinder 's #857 PR which is also in code review? It is just one line further, see https://jenkins.catrob.at/job/Paintroid/job/PR-870/8/testReport/org.catrobat.paintroid.test.espresso/MenuFileActivityIntegrationTest/Tests___Device_Tests___testSaveCopy/ but actually is already occurring long before your pull request.

@CodeChamp-SS another possibility for a cross-review of each other's tickets is #858 of @harsh-hash

**Review by ThorstenBandel — APPROVED** [review]
LGTM

**Comment by CodeChamp-SS** [comment]
> 🙏 @CodeChamp-SS could you cross-review @chbinder 's #857 PR which is also in code review? It is just one line further, see https://jenkins.catrob.at/job/Paintroid/job/PR-870/8/testReport/org.catrobat.paintroid.test.espresso/MenuFileActivityIntegrationTest/Tests___Device_Tests___testSaveCopy/ but actually is already occurring long before your pull request.

I think changes made by @chbinder in testSaveCopy should fix it.

**Comment by wslany** [comment]
Thanks!!

**Pull request merged**